internal/extsvc/github/pulls.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/sourcegraph/sourcegraph/internal/vcs/git"
 )
 
+// Redux paging information.
```

Comment:
In case you didn't know this: it's [recommended practice](https://blog.golang.org/godoc) to start code comments with the subject of the comment:

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

```suggestion
// PageInfo contains the paging information based on the Redux conventions.
```